packages/typespec-vscode/package.json

```diff
@@ -108,6 +108,90 @@
             ],
             "default": "off",
             "description": "Define whether/how the TypeSpec language server should send traces to client. For the traces to show properly in vscode Output, make sure 'Log Level' is also set to 'Trace' so that they won't be filtered at client side, which can be set through 'Developer: Set Log Level...' command."
+          },
+          "typespec.generateCode.emitters": {
+            "scope": "window",
+            "type": "array",
+            "items": {
+              "type": "object",
+              "properties": {
+                "language": {
+                  "type": "string",
+                  "enum": [
+                    ".NET",
+                    "Java",
+                    "JavaScript",
+                    "Python",
+                    "Go",
+                    "OpenAPI3",
+                    "ProtoBuf",
+                    "JsonSchema"
+                  ],
+                  "description": "Define the language the emitter will emit."
+                },
+                "package": {
+                  "type": "string",
+                  "description": "Define the emitter package.\n\nExample (with version): @typespec/http-client-csharp@1.0.0\n\nExample (without version): @typespec/http-client-csharp"
+                },
+                "sourceRepo": {
```

Comment:
user may not have repo and may want to show other information. How about name it "description" instead of "sourceRepo"?